inst/inst_CRIRES.py

```diff
@@ -109,6 +111,75 @@ def Tell(molec):
       return w_atm, f_atm
 
 
+def write_fits(wtpl_all, tpl_all, e_all, list_files, file_out):
+
+    file_in = list_files[0]
+
+    # copy header from first fits file 
+    hdu = fits.open(file_in, ignore_blank=True)
+    hdr = hdu[0].header
+
+    if len(list_files) > 1:
+        # delete parts that vary for all observations
+        del hdr['DATE-OBS']
+        del hdr['UTC']
+        del hdr['LST']
+        del hdr['ARCFILE']
+        del hdr['ESO INS SENS*']
+        del hdr['ESO INS TEMP*']
+        del hdr['ESO INS1*']
+        del hdr['ESO DET*']
+        del hdr['ESO OBS*']
+        del hdr['ESO TPL*']
+        del hdr['ESO TEL*']
+        del hdr['ESO OCS MTRLGY*']
+        del hdr['ESO ADA*']
+        del hdr['ESO AOS*']
+        del hdr['ESO SEQ*']
+        del hdr['ESO PRO DATANCOM']
+        del hdr['ESO PRO REC1 PARAM*']
+        del hdr['ESO PRO REC1 RAW*']	
+
+        for hdri in hdu:
+            hdri.header['EXPTIME'] = 0
+
+    # file creation date 
+    now = datetime.now()
+    dt_string = now.strftime("%Y-%m-%dT%H:%M:%S")
+    hdr['DATE'] = dt_string
+
+    # save raw file informations in FITS header
+    hdr.set('ESO PRO REC2 ID', 'viper_create_tpl', 'Pipeline recipe', after='ESO PRO REC1 PIPE ID')
```

Comment:
For the moment, I decided to keep the existing ESO format and comments. Nevertheless, in this regard I guess a lot more changes and additional comments will follow. Plus I need to study the document for the specifications more carefully and have to clarify with people from ESO which comments have to be used. I just didn't wanted to delay the update because of that. Maybe shift that discussions about header structures to a later point?